glean-core/src/common_metric_data.rs

```diff
@@ -45,17 +49,25 @@ impl TryFrom<i32> for Lifetime {
     }
 }
 
+/// The common set of data shared across all different metric types.
 #[derive(Default, Debug, Clone)]
 pub struct CommonMetricData {
+    /// The metric's name.
     pub name: String,
+    /// The metric's category.
     pub category: String,
+    /// List of pings to include this metric in.
```

Comment:
```suggestion
    /// List of ping names to include this metric in.
```

To disambiguate from the `PingType` objects.  (It's clear from the data type, of course, so feel free to ignore...